Sources/TuistKit/Commands/TestCommand.swift

```diff
@@ -44,6 +44,12 @@ struct TestCommand: ParsableCommand {
     )
     var configuration: String?
 
+    @Option(
+        name: .shortAndLong,
+        help: "Path where the automation project will be generated."
+    )
+    var automationPath: String?
```

Comment:
I'd agree but since there are changes between the project for testing and the one users actually use, it's quite possible there will be more issues - and without this there is no way they could debug this - it's also necessary for the acceptance tests. I can hide this with some env variable if we don't want it to be part of the command itself

Reply:
OK, sorry - I did not finish reading all the comments 😅 

[This](https://github.com/tuist/tuist/pull/2422#discussion_r571585238) should alleviate that need for users (we can print the path the project is generated to). I will still need to think about whether this will work for acceptance tests, too